kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -712,6 +721,8 @@ var ExercisePracticeView = ExerciseWrapperBaseView.extend({
         });
 
         this.listenTo(this.exercise_view, "check_answer", this.check_answer);
+
+        this.listenTo(this.exercise_view, "hint_available", this.toggle_hint_view);
```

Comment:
Disregard, I see that it's handled by [Events.trigger](http://backbonejs.org/#Events-trigger).